stylesheet.css

```diff
@@ -1,195 +1,451 @@
+  height: 14rem;
+  margin: 2rem;
 }
 
-.sum-contact{
-  border-top-left-radius: 5px;
-  border-top-right-radius: 5px;
-  border-bottom-right-radius: 5px;
-  border-bottom-left-radius: 5px;
-  width: 150px; 
-  height: 25px;
+#projects ul li figure {
+  display: table;
+  padding-bottom: 1rem;
+}
+
+#projects ul li figure img {
+  width: 12.25rem;
+  height: 10rem;
+}
+
+#projects ul li figure figcaption {
+  text-align: center;
+  width: 100%;
+  display: table-caption;
+  caption-side: bottom;
+  font-size: 0.9em;
+}
+
+/*-- open-source section --*/
+#open {
+  color: #373737;
+  text-align: center;
+}
+
+#open > ul {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  font-family: 'Roboto', sans-serif;
+}
+
+#open > ul > li {
+  border-bottom-width: medium;
+  border-bottom: 0.1rem solid lightgrey;
+  border-color: #8F8F8F;
+  padding: 1rem 1.25rem;
+  display: flex;
+  flex-direction: row;
+}
+#open > ul > li > figure {
+  display: flex;
+}
+
+#open > ul > li > figure > img {
+  float: left;
+  padding: 2rem;
+  width: 7rem;
+}
+
+#open > ul > li > figure > figcaption {
+  text-align: left;
+  align-self: center;
+  }
+
+#open > ul > li > figure > figcaption > header {
+  font-weight: bold;
+}
+
+#open > ul > li > section {
+  float: right;
+  display: flex;
+  flex-direction: row;
+  padding-left: 2rem;
+}
+
+#open > ul > li > section > img {
+  width: 6rem;
+  height: 6rem;
+  padding: 0.25rem;
+}
+
+/*-- community section --*/
+#community > ul {
+  display: flex;
+  flex-flow: row wrap;
+  font-family: 'Roboto', sans-serif;
 }
 
-.sum-main-box {
+#community > ul > li > figure {
+  display: flex;
   align-items: center;
+  padding-bottom: 2rem;
+}
+
+#community > ul > li > figure > img {
+  padding: 2rem;
+}
+
+#community > ul > li > figure > figcaption > header {
+  font-weight: bold ;
+  padding-bottom: 1rem;
+  font-size: 1.4rem;
+}
+
+#community p,
+#open p {
+  font-weight: 100;
+  line-height: 160%;
+}
+
+/*-- sum-up page --*/
+#sum-up_page > section {
+  display: flex;
+  justify-content: space-between;
+}
+
+#sum-up_page > section > section > address{
+  align-self: center;
+  font-style: normal;
+  padding-top: 1rem;
+  padding-bottom: 1rem;
+  line-height: 1.5rem;
+}
+#sum-up_page > section > section {
+  align-self: center;
+}
+
+
+#sum-up_contact {
+  display: flex;
+  flex-direction: column;
   justify-content: center;
+  align-items: center;
+}
+
+#sum-up_contact > p {
+  padding: 0.5rem;
+  line-height: 1.5rem;
   text-align: center;
 }
 
-.sum-green {
+#sum-up_contact > form > input {
+  margin-bottom: 0.5rem;
+  border: 0.01rem solid grey;
+}
+
+#sum-up_contact > footer {
+  width: 45rem;
+  padding: 1rem;
+  font-size: 0.75em;
+}
+
+.sum-contact{
+  width: 22rem;
+  height: 2rem;
+  padding: 0.5rem;
+}
+
+#sum-up_contact input.sum-up-submitbox {
+  background-color: #73e600;
+  display: block;
+  border: none;
+  margin: auto;
+  width: 6rem;
+  border-radius: 1rem;
+  color: white;
+}
+
+.sum_up-slogan {
```

Comment:
I don't like mixed `_` and `-` in class names. It's not mistake, it's about convention. If you're using everywhere dashes - be consitent in this project (there is no need here to use this mixing syntax)